typings/rsmq-promise.d.ts

```diff
@@ -0,0 +1,34 @@
+declare module 'rsmq-promise' {
```

Comment:
It would be better to update to rsmq 0.10 and remove the need for custom types + manual promisification altogether

Reply:
I have done this now, but it requires a bit of a nasty hack to make `Promise` a `bluebird` one and not native. Is this desirable?